src/main/webapp/gameDialogue.js

```diff
@@ -25,3 +25,30 @@ function addDialogueToDom(dialogue) {
   const quoteContainer = document.getElementById('dialogue-container');
   quoteContainer.innerText = dialogue;
 }
+
+
+function getImage() {  
+  fetch("/image-handler")
+      .then(response => response.text())
+      .then(message => {
+          var messageArray = message.split("\n");
```

Comment:
Maybe add a comment about what format the messageArray comes in/ what each line represents.